content/zh-cn/docs/tasks/extend-kubectl/kubectl-plugins.md

```diff
@@ -211,7 +211,8 @@ kubectl foo config
 ```
 
 ```shell
-KUBECONFIG=/etc/kube/config kubectl foo config
+KUBECONFIG=/etc/kube/config 
```

Comment:
we don't need this change, one line is fine. `export` is required if split into 2 lines

Reply:
I got it.